src/libraries/System.Private.CoreLib/src/System/Reflection/NullabilityInfo.cs

```diff
@@ -8,7 +8,12 @@ namespace System.Reflection
     /// <summary>
     /// A class that represents nullability info
     /// </summary>
-    public sealed class NullabilityInfo
+#if NETCOREAPP
```

Comment:
```suggestion
#if NET
```
There was a recent replace of NETCOREAPP -> NET https://github.com/dotnet/runtime/pull/101693.

Reply:
Fixed.